### PR TITLE
JS Client V1

### PR DESCRIPTION
**Pull request by victorchimakanu**
Updated readme doc and api.js code

### CODE REVIEW

File: javascript/README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
+
 # BagelDB JavaScript Client 🥯
```
Comment (BidhanRoy) [inline javascript/README.md:2]: BagelDB -> Bagel

**Comment by victorchimakanu** [comment]
Not passing JS Standard (pull_request) . Looking into that

**Comment by victorchimakanu** [comment]
All checks have passed

**Comment by victorchimakanu** [comment]
Testing:

- I used node to run my files 
- I installed the following dependencies :

1. Axios : `npm install axios`
2. Node-Fetch: `npm install node-fetch`
3. Form data: `npm install form-data`
4. uuid: `npm install uuid`
5. Buffer: `npm install buffer`

- I ran npx standard --fix to pass linter checks

**Comment by victorchimakanu** [comment]
**Testing Functions**

To test all functions described in the README, I installed the BagelDB beta node package and the following dependencies:

- Axios: `npm install axios`
- Node-Fetch: `npm install node-fetch`
- Form Data: `npm install form-data`
- UUID: `npm install uuid`
- Buffer: `npm install buffer`

I created functions for `create_asset`, `get_asset_by_id`, `get_all_assets`, and `delete_asset`. These API class methods were added to the `api.js` file in the `src` directory. To test these functions, follow these steps:

1. Install the BagelDB beta node package and update `src/api.js` in your `node_modules` folder with the `api.js` file provided in the JavaScript client.
&nbsp; 
2. In the BagelDB beta folder inside your `node_modules`, update it with the `settings.js`, `config.js`, and `bageldb.js` files I updated in the client. Note that:
&nbsp; 
   - For `settings.js` and `config.js`, I made changes to how the files were exported.
   - For the changes in`bageldb.js`. Initially, it used `module.exports` to export files, but I updated it to ES6 standards, using an`export` statement and aliases. 
   - Significant time was spent ensuring the code adhered to ES6 standards and worked smoothly.
   &nbsp; 
3. The JavaScript code was not based on the recent ES6 standard—it used `require` instead of `import` for modules. 
I updated that. In your `package.json` file inside your `node_modules` folder, add this line:
&nbsp; 
   ```json
   "type": "module",
   ```
   It should be underneath `"main": "bageldb.js",`.
   &nbsp; 
4. Ensure you have npm installed.

To run the files:

- **Create Asset:**
  Refer to the create asset instructions in the README. Copy the code and run with `node filename.js`.

- **Get Asset by ID:**
  Refer to the get asset by ID instructions in the README. Copy the code and run with `node filename.js`.

- **Get All Assets:**
  Refer to the get all assets instructions in the README. Copy the code and run with `node filename.js`.

- **Delete Assets:**
  Refer to the delete asset instructions in the README. Copy the code and run with `node filename.js`.

**Linter Checks:**
I ran `npx standard --fix` to pass linter checks.

**Pull request merged**